packages/api/types.ts

```diff
@@ -0,0 +1,2 @@
+// Abstraction so that packages don't have to add `@agreeo/db` as a dependency just for these types
```

Comment:
If we import the types as `import type { SomeType }` does it still require the real dependency?

Reply:
well the enum's aren't types so that doesn't work